import-service/package.json

```diff
@@ -11,13 +11,13 @@
   "license": "ISC",
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.12.1",
-    "aws-sdk-mock": "^5.1.0",
     "jest": "^26.6.3",
     "serverless-webpack": "^5.3.5",
     "webpack": "^5.4.0"
   },
   "dependencies": {
     "aws-sdk": "^2.792.0",
```

Comment:
Please move it to dev-dependencies and reduce the weight of your function

Reply:
It doesn't affect the weight of the function because in any case in will be included in the build because of the import statement